### PR TITLE
Adjust translation box title of content form

### PR DESCRIPTION
**Pull request by lunars97**
### Short description

Remove the unnecessary word "another" from the title of the automatic translation text of **minor edit** box so that it was cleaner

### Proposed changes

- Adjust the German title of the block to "Auswirkungen auf Übersetzungen"

### Resolved issues

Fixes: #2690 

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

File: integreat_cms/cms/templatetags/content_filters.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def minor_edit_label(region: Region, language: Language) -> Promise:
     if language_node.is_leaf():
         return _("Implications on the translation status")
     if language_node.is_root():
-        return _("Implications on other translations")
+        return _("Implications on translations")
     return _("Implications on the status of this and other translations")
```
Comment (timobrembeck) [inline integreat_cms/cms/templatetags/content_filters.py:104]: @dkehne I assume that you also wanted to make the heading more clear for the languages other than the default language (e.g. English, Arabic etc.), right?

At the moment, this looks like this:

Screenshot 2024-03-26 at 17-55-22 Integreat Redaktionssystem

I guess this should probably also be changed to "Implications on translations"?

File: integreat_cms/cms/templatetags/content_filters.py [hunk 2]
```diff
@@ -100,7 +100,7 @@ def minor_edit_label(region: Region, language: Language) -> Promise:
     if language_node.is_leaf():
         return _("Implications on the translation status")
     if language_node.is_root():
```
Comment (timobrembeck) [inline integreat_cms/cms/templatetags/content_filters.py:102]:
```suggestion
```
Then we could simplify this `if` statement.

File: integreat_cms/cms/templatetags/content_filters.py [hunk 3]
```diff
@@ -100,7 +100,7 @@ def minor_edit_label(region: Region, language: Language) -> Promise:
...
-        return _("Implications on other translations")
+        return _("Implications on translations")
```
Comment (timobrembeck) [inline integreat_cms/cms/templatetags/content_filters.py:103]:
```suggestion
    return _("Implications on translations")
```

File: integreat_cms/cms/templatetags/content_filters.py (hunk 1 above)
Comment (timobrembeck) [inline integreat_cms/cms/templatetags/content_filters.py:104]:
```suggestion
```

**Review by timobrembeck — CHANGES_REQUESTED** [review]
@lunars97 Awesome, thanks a lot for your first PR! :tada: :muscle: 

In general it looks very good! :+1:
I just assume that we could simplify the box heading for non-root languages as well.

File: integreat_cms/cms/templatetags/content_filters.py (hunk 1 above)
Comment (lunars97) [reply integreat_cms/cms/templatetags/content_filters.py:104]: yep, thank you for your review. I will work on the rest :)

**Review by MizukiTemma — APPROVED** [review]

**Review by timobrembeck — COMMENTED** [review]
Sorry about the long delay!
Could you squash your commits into one? Then I think this is ready to be merged! :tada:

**Review by timobrembeck — APPROVED** [review]

**Pull request merged**